en/file_formats/geofence.md

```diff
@@ -0,0 +1,80 @@
+# GeoFence Plan File Format (Mission Protocol)
+
+GeoFence plan files are formatted using JSON, and can defined a cyclindrical geofence using parameters <!-- (autopilot-specific?) --> and/or an arbitrary polygon.
+
+<!--  doc originates from *QGroundControl Dev Guide*: https://dev.qgroundcontrol.com/en/file_formats/fence.html -->
+
+```
+{
+    "fileType": "GeoFence",
+    "groundStation": "QGroundControl",
+    "parameters": [
+        {
+            "compId": 1,
+            "name": "FENCE_ENABLE",
```

Comment:
@DonLakeFlyer All these params are for PX4 geofence - ie cylindrical geofence defined using parameters.  What happens with ArduPilot and others?
Or is this PX4 specific (in which case, presumably not part of the standard?)

Reply:
It looks like you are running an old build?